service-worker/foreign-fetch/remote-server.js

```diff
@@ -0,0 +1,38 @@
+// This is the web server implementation for the "random number API".
+// It is currently deployed to https://foreign-fetch-demo.appspot.com
+
+const express = require('express');
```

Comment:
Sure, I'll add for the benefit of folks who are poking around the source tree. Though I'm assuming that most folks will see `remote-server.js` inline in the templatized web page.